packages/template-retail-react-app/app/pages/cart/partials/cart-secondary-button-group.jsx

```diff
@@ -100,20 +102,29 @@ const CartSecondaryButtonGroup = ({
                     </Button>
                 </ButtonGroup>
                 <Flex alignItems="center">
-                    <Checkbox spacing={2} isReadOnly={true}>
+                    <Checkbox
+                        name={`gift-checkbox-${variant.itemId}`}
+                        spacing={2}
+                        isChecked={isAGift}
+                        onChange={(e) => {
+                            const checked = e.target.checked
+                            onIsAGiftChange(variant, checked)
+                        }}
```

Comment:
For the checkbox, we'd like the following to happen:
- check/uncheck the checkbox using both space and enter (only space works right now)
- remove the `aria-hidden` attribute on the underlying span element

Unfortunately, I don't believe there's an easy way to do this through the component as there doesn't seem to be anything in the docs to support these use cases. An alternative would be to use `document.querySelector` to grab the element and modify it but that seems kind of hacky. Wondering if there's a cleaner solution 🤔

Reply:
Just space is fine - that's what's supported natively. (e.g. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox)

Reply:
@joeluong-sfcc Space should be fine as @wjhsf pointed out above

Regarding the `aria-hidden`, we are currently using Chakra Checkbox, and they are using a span element to construct the checkbox's UI. It is not necessary to remove the aria-hidden since it is not the problem. The problem was the checkbox never worked in the first place. I've fixed it and it is working along with the API. The screen reader is able to pick up the status of the checkbox.